ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/PullPhysicalPlanBuilder.java

```diff
@@ -191,6 +197,16 @@ private SelectOperator translateFilterNode(final QueryFilterNode logicalNode) {
     return new SelectOperator(logicalNode, logger);
   }
 
+  private LimitOperator translateLimitNode(final QueryLimitNode logicalNode) {
+    final ProcessingLogger logger = processingLogContext
```

Comment:
You can get rid of this unless you're going to use it.  I don't think we actually use this for pull queries, except where we share APIs with persistent queries and must, namely the SelectValueMapper which is shared.  The reason is that we can err immediately and surface that error whereas persistent queries cannot.